.github/workflows/build.yml

```diff
@@ -152,11 +152,12 @@ jobs:
         if: github.ref == 'refs/heads/master'
 
         run: |
-            git config --local user.name ${{ github.actor }}
+            git config user.name 'github-actions[bot]'
+            git config user.email 'github-actions[bot]@users.noreply.github.com'
```

Comment:
This will show a gh-icon associated to this user, like this: 

![2023-07-25_16-06](https://github.com/input-output-hk/formal-ledger-specifications/assets/2070003/ecd27307-f7b0-478c-a563-cdd60e64b563)